docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+
+To deploy the connected registry module using the Azure CLI, save the following deployment manifest locally as a `.json` file. 
+
+[!IMPORTANT] In the bellow deployment manifest, the IP address `10.16.7.4` is the IP address of the device hosting parent connected registry. Make sure you replace this IP address with the one your parent device uses.
+
+```json
+{
+    "modulesContent": {
+        "$edgeAgent": {
+            "properties.desired": {
+                "modules": {
+                    "connected-registry": {
+                        "settings": {
+                            "image": "10.16.7.4/acr/connected-registry:0.2.0",
+                            "createOptions": "{\"HostConfig\":{\"Binds\":[\"/home/azureuser/connected-registry:/var/acr/data\",,\"/usr/local/share/ca-certificates:/usr/local/share/ca-certificates\",\"/etc/ssl/certs:/etc/ssl/certs\"]}}"
+                        },
+                        "type": "docker",
+                        "env": {
+                            "ACR_REGISTRY_CONNECTION_STRING": {
+                                "value": "ConnectedRegistryName=myconnectedmirror;SyncTokenName=myconnectedmirror-sync-token;SyncTokenPassword=s0meCoMPL3xP4$$W0rd001!@#;ParentGatewayEndpoint=10.16.7.4;ParentEndpointProtocol=https"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "version": "1.0"
+                    },
+                    "IoTEdgeApiProxy": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-api-proxy:latest",
+                            "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"443/tcp\": [{\"HostPort\": \"443\"}]}}}"
+                        },
+                        "type": "docker",
+                        "version": "1.0",
+                        "env": {
+                            "NGINX_DEFAULT_PORT": {
+                                "value": "443"
+                            },
+                            "CONNECTED_ACR_ROUTE_ADDRESS": {
+                                "value": "connectedRegistry:8080"
+                            },
+                            "NGINX_CONFIG_ENV_VAR_LIST": {
+                                    "value": "NGINX_DEFAULT_PORT,BLOB_UPLOAD_ROUTE_ADDRESS,CONNECTED_ACR_ROUTE_ADDRESS,IOTEDGE_PARENTHOSTNAME,DOCKER_REQUEST_ROUTE_ADDRESS"
+                            },
+                            "DOCKER_REQUEST_ROUTE_ADDRESS": {
+                                "value": "registry:5000"
+                            },
+                            "BLOB_UPLOAD_ROUTE_ADDRESS": {
```

Comment:
I think we can BLOB_UPLOAD_ROUTE_ADDRESS?

Reply:
This is required since it is included in the current nginx config. If you don't have it, the api proxy will throw error like unknown "blob_upload_route_address" variable.

Reply:
Removed DOCKER env variable only.